packages/bpk-component-infinite-scroll/src/withInfiniteScroll-test.flow.js

```diff
@@ -65,13 +65,13 @@ const InfiniteList = withInfiniteScroll(List);
 
 (() => (
   <React.Fragment>
-    {/* $ExpectError (aria-label is required) */}
+    {/* $FlowExpectedError[prop-missing] */}
```

Comment:
Not your fault mainly just a pain for this as the change to the error expect doesn't give more context any more 😞 e.g. this now just says `prop-missing` vs `aria-label is required`

Reply:
Yeah I noticed that too, bit of a pain